CHANGELOG.md

```diff
@@ -13,6 +14,7 @@
   * `operator.strimzi.io/generation` (used internally only - replaced by `strimzi.io/generation`)
   * `operator.strimzi.io/delete-pod-and-pvc` (use `strimzi.io/delete-pod-and-pvc` instead)
   * `operator.strimzi.io/manual-rolling-update` (use `strimzi.io/manual-rolling-update` instead)
+* External logging configuration has changed. `spec.logging.name` is deprecated. Moved to `spec.logging.valueFrom.configMapKeyRef.name`. Key in the `ConfigMap` is configurable via `spec.logging.valueFrom.configMapKeyRef.key`.
```

Comment:
Maybe you can add a simple example of deprecated and new - I think that would help to understand it better. That would serve also as a conversion guide in a way.